tests/unit_tests/test_artanh.py

```diff
@@ -0,0 +1,51 @@
+import json
+import math
+
+import pytest
+
+from tests.utils import load_process_code, run_process
+
+
+@pytest.fixture
+def artanh_process_code():
+    return load_process_code("artanh")
+
+
+@pytest.mark.parametrize(
+    "example_input,expected_output",
+    [
+        ({"x": 0}, 0),
+        ({"x": 0.5}, 0.549306144334055),
+        (
+            {"x": 1},
+            None,
+        ),  # JS returns Infinity, but Python doesn't recognize JS's Infinity
+        ({"x": 2}, None),  # JS returns NaN, but Python doesn't recognize JS's NaN
```

Comment:
akward. it works the other way around  e.g passing float('nan') as argument. fftr

Reply:
Yup, it's interesting. I think it would be good to still check later if it's possible to correctly recognize this in python. But let's merge this.